### PR TITLE
adds an add review modal as well as a large number of styling changes

### PR DESCRIPTION
**Pull request by APolk10**
See ratingsTile.jsx and addReview.jsx

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -230,11 +229,87 @@ header {
...
+#radioTable2 {
+  display: table;
+  width: 65vw;
+  border: 1px solid black;
+  border-collapse: collapse;
+}
+
+td { /*this may cause issues but I can go back later and fix */
+  border: 1px solid black;
+  text-align: center;
```
Comment (tanner-white) [inline public/styles.css:288]: what issues do you see this causing?

Comment (APolk10) [reply public/styles.css:288]: I did a lot of CSS changes and had a prior merge conflict with the QA section. It is possible there is a conflict but unlikely.

Comment (tanner-white) [reply public/styles.css:288]: What merge conflict were you experiencing?

Comment (APolk10) [reply public/styles.css:288]: There was an issue with some overlap in the CSS file between Jacob's modal and mine. Merge conflicts should be resolved now.

Comment (tanner-white) [reply public/styles.css:288]: Got it. Thanks for clarifying!

**Review by tanner-white — APPROVED** [review]

**Pull request merged**